### DISCUSSION TITLE
Welcome to avr8js-electron-playground Discussions!

### DISCUSSION THREAD

**Discussion by arcostasi** [arcostasi/avr8js-electron-playground#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Areopagitics** [comment]
Hello @arcostasi, I'm very interested in this project for the quick testing and prototyping. Will you make it MIT licensed?

**Comment by arcostasi** [comment]
Hello @Areopagitics, Yes! I’m happy to keep this project MIT licensed. The repository already declares MIT in package.json, and the main dependencies I’m using are under permissive licenses compatible with MIT like https://github.com/wokwi/avr8js and https://github.com/wokwi/wokwi-elements. I’ll also add a proper LICENSE file at the repository root so it’s explicit on GitHub as well. Thanks a lot for the interest!
